app/Http/Controllers/ProfileController.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\PublisherProfileRequest;
+use App\Http\Requests\UserProfileRequest;
+use App\Models\Publisher;
+use App\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Config;
+
+class ProfileController extends Controller
+{
+	public function index()
+	{
+		if (Auth::check()) {
+			if (Auth::user()->role == Config::get('role.user')) {
+				$user = Auth::user()->user;
```

Comment:
```suggestion
				$user = Auth::user();
```
c thấy như này là lấy được đối tượng người dùng rồi mà nhỉ